src/markdown-pages/docs/ecosystem/galasa-cli.md

```diff
@@ -0,0 +1,153 @@
+
+The following example command selects specified tests (`SimBankIVT` and `BasicAccountCreditTest`) from the `dev.galasa.simbank.tests` test package and adds these tests to the `test.yaml` portfolio. 
+
+```
+galasactl runs prepare
+                        --portfolio test.yaml
+                        --class dev.galasa.simbank.tests/SimBankIVT
+                        --class dev.galasa.simbank.tests/BasicAccountCreditTest
+```
+
+### Setting test-specific overrides
+
+Specifying overrides is useful if you want to run a set of tests against a particular configuration without changing the test code. For example, you might have multiple versions of software that you need to test. How can you do that without changing the test code? The answer is to use override properties. If you are running tests locally, you can set overrides properties by editing your `Overrides Properties` file. If you are running tests in an ecosystem, you can use the `--override` parameter in the Galasa CLI. Note that overrides that are specified in the Galasa CLI take precedence over those set in your local `Overrides Properties` file.
+
+The following example creates a portfolio called `test.yaml` that contains tests from the `BestSoFar` test stream where the test package is `dev.galasa.simbank.tests`.
+
+All the tests in the `test.yaml` portfolio will run on the z/OS LPAR `MV2C` in the `PLEX2` cluster when the `galasactl runs submit --portfolio test.yaml` command is run, regardless of the LPAR and cluster that is specified in the CPS properties file.
+
+```
+galasactl runs prepare
+                        --portfolio test.yaml
+                        --stream BestSoFar
+                        --package dev.galasa.simbank.tests
+                        --override zos.default.lpar=MV2C
+                        --override zos.default.cluster=PLEX2
+```
+
+### Building a portfolio over multiple selections and overrides
+
+In the following example, the first command creates a portfolio called `test.yaml` that contains tests from the `BestSoFar` test stream where the test package name is `dev.galasa.simbank.tests`. The second command adds more tests to the `test.yaml` portfolio from the `BestSoFar` test stream where the test package name is `dev.galasa.simbank.tests.two`. 
+
+All the tests in the `test.yaml` portfolio from `dev.galasa.simbank.tests` will run on the z/OS LPAR `MV2C` in the `PLEX2` cluster. The tests from `dev.galasa.simbank.tests.two` will run on the z/OS LPAR `MV2D` in the `PLEX2` cluster when the `galasactl runs submit --portfolio test.yaml` command is run.
+
+```
+galasactl runs prepare 
+                        --portfolio test.yaml
+                        --stream BestSoFar
+                        --package dev.galasa.simbank.tests
+                        --override zos.default.lpar=MV2C
+                        --override zos.default.cluster=PLEX2
+galasactl runs prepare
+                        --portfolio test.yaml
+                        --append
```

Comment:
not sure you highlight "--append" above,  although it is implicit in the command